src/index.js

```diff
@@ -56,6 +56,7 @@ export default (elementId, options = {}) => {
 
     playerAPI = YouTubePlayer.promisifyPlayer(playerAPIReady);
     playerAPI.on = emitter.on;
+    playerAPI.off = emitter.off;
```

Comment:
Docs should simply declare that such method exists. I am referring to the README. https://github.com/gajus/youtube-player#events